game.cpp

```diff
@@ -1189,12 +1194,13 @@ void Game::WriteReport()
 	MakeFactionReportLists();
 	CountAllSpecialists();
 
-	int ** citems = new int* [factionseq];
```

Comment:
this memory was getting leaked previously.  Since we only need this array if we are colleting faction statistics, I cleaned it up a bit.